project_generator_definitions/mcu/st/stm32f401xe.yaml

```diff
@@ -37,6 +37,8 @@ tool_specific:
             - null
             FlashDriverDll:
             - UL2CM3(-S0 -C0 -P0 -FD20000000 -FC1000 -FN1 -FF0STM32F4xx_512 -FS08000000 -FL080000 -FP0($$Device:STM32F401RETx$CMSIS\Flash\STM32F4xx_512.FLM))
+            TargetDriverDllRegistry_SetRegEntry_Name:
+            - -U0672FF484849785087111542 -O78 -S0 -C0 -A0 -N00("ARM CoreSight SW-DP") -D00(2BA01477) -L00(0) -TO18 -TC10000000 -TP21 -TDS8007 -TDT0 -TDC1F -TIEFFFFFFFF -TIP8 -FO7 -FD20000000 -FC1000 -FN1 -FF0STM32F4xx_512.FLM -FS08000000 -FL080000 -FP0($$Device:STM32F401RETx$CMSIS\Flash\STM32F4xx_512.FLM)
```

Comment:
> Unfortunately a nTsel is not enough i.e. without a SetRegEntry part the uvoptx file doesn't work.
 
we can generate at least that info, the minimum set. This means we use debugger settings that are generic, and progen definitions are without any changes, that would be great.